zulip_bots/zulip_bots/bots/salesforce/test_salesforce.py

```diff
@@ -0,0 +1,196 @@
+from zulip_bots.test_lib import BotTestCase, read_bot_fixture_data
+import simple_salesforce
+from simple_salesforce.exceptions import SalesforceAuthenticationFailed
+from contextlib import contextmanager
+from unittest.mock import patch
+from typing import Any, Dict
+import logging
+
+
+@contextmanager
+def mock_salesforce_query(test_name: str, bot_name: str) -> Any:
+    response_data = read_bot_fixture_data(bot_name, test_name)
+    sf_response = response_data.get('response')
+
+    with patch('simple_salesforce.api.Salesforce.query') as mock_query:
+        mock_query.return_value = sf_response
+        yield
+
+
+@contextmanager
+def mock_salesforce_auth(is_success: bool) -> Any:
+    if is_success:
+        with patch('simple_salesforce.api.Salesforce.__init__') as mock_sf_init:
+            mock_sf_init.return_value = None
+            yield
+    else:
+        with patch(
+            'simple_salesforce.api.Salesforce.__init__',
+            side_effect=SalesforceAuthenticationFailed(403, 'auth failed')
+        ) as mock_sf_init:
+            mock_sf_init.return_value = None
+            yield
+
+
+@contextmanager
+def mock_salesforce_commands_types() -> Any:
+    with patch('zulip_bots.bots.salesforce.utils.commands', mock_commands), \
+            patch('zulip_bots.bots.salesforce.utils.object_types', mock_object_types):
+        yield
+
+
+mock_config = {
+    'username': 'name@example.com',
+    'password': 'foo',
+    'security_token': 'abcdefg'
+}
+
+help_text = '''Salesforce bot
+This bot can do simple salesforce query requests
+Commands:
+**find contact <name> [arguments]**: finds contacts
+**find top opportunities <amount> [arguments]**: finds opportunities
```

Comment:
In `doc.md`, you mention
> +It can also show details about any Saleforce links that you post.

Can you add a respective command to this help page?